index.html

```diff
@@ -20,20 +20,192 @@
             </div>
         </div>
         </div>
-    </header>
-    <main>
+
         <section>
-            <h1>Nwannah  <br>Cynthia</h1>
+            <h2>Nwannah  <br>Cynthia</h2>
             <p>Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works. <br> 
                 If you like what you see and have a project you need coded, don’t hesitate and contact me</p>
             <div class="bot">
             <input type="button" value="Start Collaboration ">
         </div>
-
     </section>
+    </header>
+    
+    <main>    
     <div class="sectiontwo">
-        <!-- <img src="./asset/shape.svg" alt="A joined shape image"> -->
+        <img src="./asset/shape.svg" alt="">
+    </div>
+    <div class="pro">
+        <h1>projects</h1>
+    </div>
+    <div id="section-three">
+        <div class="lap">
+            <h1>welcome</h1>
+        <img src="./asset/laptop-image.png" alt="">
+    </div>
+    </div>
+    <div class="section-threeb">
+        <div class="main-se">
+        <div class="one"><p>My Project Portfolio</p></div>
+        <div class="two"><button type="button">HTML/CSS</button></div>
+        <div class="three"><button type="button">Ruby on Rails</button></div>
+        <div class="four"><button type="button">Javascript</button></div>
+        <div class="five"><p>See this project <img src="./asset/arrowright.png" alt=""></p></div>
+    </div>
+    </div>
+    <div id="section-threeb">
+        <div class="lap">
+        <img src="./asset/laptop2image.png" alt="">
+    </div>
+    </div>
+    <div class="section-threeb">
+        <div class="main-se">
+        <div class="one"><p>My Project Portfolio</p></div>
+        <div class="two"><button type="button">HTML/CSS</button></div>
+        <div class="three"><button type="button">Ruby on Rails</button></div>
+        <div class="four"><button type="button">Javascript</button></div>
+        <div class="five"><p>See this project <img src="./asset/arrowright.png" alt=""></p></div>
+    </div>
     </div>
+<div id="section-threec">
+    <div class="lap">
+    <img src="./asset/laptopfour.png" alt="">
+</div>
+</div>
+<div class="section-threeb">
+    <div class="main-se">
+    <div class="one"><p>My Project Portfolio</p></div>
+    <div class="two"><button type="button">HTML/CSS</button></div>
+    <div class="three"><button type="button">Ruby on Rails</button></div>
+    <div class="four"><button type="button">Javascript</button></div>
+    <div class="five"><p>See this project <img src="./asset/arrowright.png" alt=""></p></div>
```

Comment:
- [ ] Please make sure that the list of technologies on all your projects is wrapped in a `ul` tag and each technology wrapped in a `li` tag. It is  good practice for lists of similar items 👍

Reply:
@Lucas-Masaba should I remove the display grid. the mention I should use both display grid and flex in this project . should I remove the grid and replace it with ul and li

Reply:
@cynthia-nwannah , sorry for the late reply. I've been busy lately. In this exercise, you are required to use both grid and flex.